docker-compose.yaml

```diff
@@ -0,0 +1,12 @@
+version: '2'
```

Comment:
I can appreciate the value of adding a Dockerfile, but seeing as the server application is one application, I cannot see any use to including a docker-compose file as well. Please could you remove this file?

Reply:
This was simply to make it easier to manage (startup, stop, etc).  For folks newer to Docker, it seemed like an easier way to test things out.  I can certainly remove it if you prefer that.

Reply:
Updated in PR.